feature-toggle-api/templates/tests/test-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Shouldn't this be a job instead of a pod?

Reply:
not sure this is important - i think it can be either....but i will check